data/settings/src/main/java/com/google/jetpackcamera/settings/model/CameraAppSettings.kt

```diff
@@ -19,16 +19,17 @@ package com.google.jetpackcamera.settings.model
 /**
  * Data layer representation for settings.
  */
-data class Settings(
+data class CameraAppSettings(
     val default_front_camera : Boolean,
     val dark_mode_status : DarkModeStatus,
-    val field : String = "todo"
+    val flash_mode_status : FlashModeStatus
 )
 
 
-fun getDefaultSettings(): Settings{
-    return Settings(
+fun getDefaultSettings(): CameraAppSettings{
```

Comment:
nit / optional 
You could also provide default values in the data class and create a constant `DEFAULT_CAMERA_APP_SETTINGS`

```
data class CameraAppSettings(
  val default_front_camera: Boolean = false,
  val dark_mode_status: DarkModeStatus = DarkModeStatus.SYSTEM,
  val flash_mode_sttatus: FlashModeStatus = FlashModeStatus.OFF
)

const val DEFAULT_CAMERA_APP_SETTINGS = CameraAppSettings()
```